packages/docs/guide/advanced/router-view-slot.md

```diff
@@ -0,0 +1,75 @@
+# RouterView slot
+
+The RouterView component exposes a slot that can be used to render the route component:
+
+```vue-html
+<router-view v-slot="{ Component }"
+  <component :is="Component" />
+</router-view>
+```
+
+The code above is equivalent to using `<router-view />` without the slot, but the slot provides extra flexibility when we want to work with other features.
+
+## Passing props and slots
+
+We could use the slot to pass props or slots to the route component:
+
+```vue-html
+<router-view v-slot="{ Component }"
+  <component :is="Component" some-prop="a value">
+    <p>Some slotted content</p>
+  </component>
+</router-view>
+```
+
+In practice, this usually isn't something you would want to do, as the route components would all need to use the same props and slots. See [Passing Props to Route Components](/guide/essentials/passing-props) for other ways to pass props.
```

Comment:
```suggestion
In practice, this usually isn't something you would want to do, as the route components would **all need to use the same props and slots**. See [Passing Props to Route Components](/guide/essentials/passing-props) for other ways to pass props.
```

Reply:
I've added the bold text and also updated the link to use a relative path.